### PR TITLE
Update to eSignature - v2.1-22.4.02.00

### PR DESCRIPTION
**Pull request by nathany-copia**
* [x] OpenAPI generator 6.4.0 (ignore the 1st commit when reviewing)
* [x] as-is update
* [x] rename number to docuSignNumber
* [x] rename date to docuSignDate
* [x] Update instructions
* [x] Testing this out beyond running the test suite

ref: #33
ref: https://github.com/docusign/OpenAPI-Specifications/pull/72

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,20 +141,25 @@ the API client for those users and refresh the access tokens.
...
-java -jar "openapi-generator-cli-6.2.1.jar" generate -i "esignature.rest.swagger-v2.1.json" -g "elixir" -o "/tmp/elixir_api_client"
+openapi-generator generate -i "esignature.rest.swagger-v2.1.json" -g "elixir" -o "/tmp/elixir_api_client"
 rm -rf lib/docusign/api/*
 rm -rf lib/docusign/model/*
 cp -rf /tmp/elixir_api_client/lib/docu_sign/api/* lib/docusign/api
 cp -rf /tmp/elixir_api_client/lib/docu_sign/model/* lib/docusign/model
 mix format
 ```
 
+NOTE: To minimize differences, also trim trailing whitespace by replacing ` +$` with nothing across all files.
```
Comment (nathany-copia) [inline README.md:161]: I imagine there's a cross-platform way to do this on the command-line as part of the bash steps above.

Comment (nathany-copia) [reply README.md:161]: (It seemed like some files had trailing spaces trimmed and others didn't.... so 🤷🏻 )

Comment (neilberkman) [reply README.md:161]: > I imagine there's a cross-platform way to do this on the command-line as part of the bash steps above.

We can try this:

```
find . -type f -name "*.ex" -exec sed -i '' 's/[[:space:]]\+$//' {} \;
```

Comment (nathany-copia) [reply README.md:161]: Thanks. I tried it on macOS from a fresh copy, but still have trailing spaces (probably more than before).

Alternatively, we could just commit the trailing spaces as-is and not worry about it 🤷🏻

**Comment by nathany-copia** [comment]
My use of this library is minimal so far -- just sending an email from a template. It still works for me with this update, but maybe other users are willing to try it out?

**Review by neilberkman — APPROVED** [review]
@nathany-copia thanks for your work on this!

**Pull request merged**